client/components/info-popover/index.jsx

```diff
@@ -50,6 +50,8 @@ export default class InfoPopover extends Component {
 
 	handleClick = event => {
 		event.preventDefault();
+		event.stopPropagation();
```

Comment:
Don't think there exists a case when we want both the popover to show and to submit another action. :)